python/cuml/cuml/cluster/kmeans.pyx

```diff
@@ -196,6 +196,18 @@ class KMeans(UniversalBase,
     <http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html>`_.
     """
 
+    _hyperparam_interop_translator = {
+        "init": {
+            # k-means++ would work, but setting it explicitly changes the configuration
+            # of the estimator compared to not specifying it. So we explicitly translate
+            # it to the default value.
+            "k-means++": "scalable-k-means++",
+        },
+        "algorithm": {
+            "elkan": "NotImplemented",
```

Comment:
Something to consider is whether we want to dispatch here. `elkan` afaik is meant for efficiency and not different results, so perhaps we would be better off keeping changing the value to lloyd and using the GPU. What do you think?

Reply:
Removed it for now.

In general I'd start with only accelerating things that we'd accelerate in "strict" mode. Allowing more things at a later time seems like the safer thing to do compared to allowing as much as possible now when we have no experience of what users do with the accelerator and what problems they run in to.